angular/src/app/constants/formAccount.ts

```diff
@@ -22,7 +22,7 @@ export const SIGN_UP_FIELDS = [
     label: 'Password',
     type: 'password',
     errorMessage:
-      'El password es requerido y debe contener al menos 1 mayúscula, 1 minúscula, 1 número y 1 caracter',
+      'La password es requerido y debe contener al menos 1 mayúscula, 1 minúscula, 1 número y 1 caracter',
```

Comment:
If you use the pronoun **la**, change 'requerido' to 'requerida', or better change 'password' to 'contraseña'.
```suggestion
      'La contraseña es requerida y debe contener al menos 1 mayúscula, 1 minúscula, 1 número y 1 carácter',
```